packages/visualizations/addon/components/navi-visualizations/table.js

```diff
@@ -47,6 +47,11 @@ export default Ember.Component.extend({
    */
   selectedSubtotal: computed.readOnly('options.showTotals.subtotal'),
 
+  /**
+   * @property {String} editing
+   */
+  editing: computed.readOnly('options.editing'),
```

Comment:
I think it is better convert `isEditingVisualization ` to utilize `vistualization.options.editing`, since `vistualization.options.editing` is more global configuration, otherwise, we need to pass value around

Reply:
This is the first time we are actually adding the configuration to the visualization. Since the trigger for this lies outside this component, It would be better to pass in a value.

Reply:
do you mean
```
{{navi-visualizations/table
  editing= vistualization.options.editing
}}
```?

Reply:
I mean add `isEditingVisualization` [here](https://github.com/yahoo/navi/blob/master/packages/reports/addon/templates/components/report-view.hbs#L21)
```hbs
{{report-visualization
            classNames='report-view__visualization'
            isEditingVisualization=isEditingVisualization
            report=report
            response=response
            container=this
            annotationData=annotationData
            onUpdateReport=(pipe
                (route-action 'onUpdateReport')
                (route-action 'validate' report)
                (route-action 'runReport' report)
                (route-action 'setHasReportRun' true)
            )
        }}
```